### ISSUE TITLE
GitHub Pages link in README is broken

### ISSUE THREAD

**Issue by noelleleigh** [gytdau/tome#13]
`http://gytdau.github.io/tome/` redirects to `http://tome.gytdau.com/` which throws an `ERR_NAME_NOT_RESOLVED` in Chrome.

**Comment by gytdau** [comment]
Guess that broke when I was changing over domains. I fixed it now, but it might take a little while to propagate. Thanks for pointing it out!

**Comment by noelleleigh** [comment]
The page that the link on the README redirects to seems to be inactive.

**Comment by gytdau** [comment]
Hey, super sorry about that one, I'll fix it when I get home today!

**Comment by gytdau** [comment]
Hey @n-leigh, it's online now at tome.gytis.co. 

Out of curiosity, how did you find this project? Are you just interested or did you find some use case? I'm asking because it's only kind of lying around right now and some of it is kinda broken, but I might end up trying to take the idea further.

**Comment by noelleleigh** [comment]
I found it a while ago, and fixed its entry on esolangs.org to point to the right repo. I checked in again while just browsing around and noticed the broken link. 

I think it could be useful to introduce someone to the basics of programming logic without having to use terse C-like syntax. 

Also, I would also suggest that you use HTTPS for that site to ensure visitors can’t be hijacked by a third party.
